components/statsTracking/functions.hpp

```diff
@@ -1,5 +1,9 @@
 class statsTracking
 {
     file = "components\statsTracking";
-    class addStatsEventHandlersToClass {};
+    class addStatsEventHandlersToClass {
+    };
+    class handleFiredEvent
+    {
+    };
```

Comment:
no reason to spread these brackets out is there

```suggestion
    class addStatsEventHandlersToClass {};
    class handleFiredEvent {};
```